docs/source/masked.rst

```diff
@@ -1,11 +1,291 @@
+Discusses why certain decisions were made (e.g. requiring masks to match for binary/reduction operations),
+differences with NumPy's :class:`MaskedArray`, and reduction semantics.
+
+Supported Operators
++++++++++++++++++++
+
+Unary Operators
+---------------
+
+Unary operators are operators that only contain only a single input.
+Applying them to MaskedTensors is relatively straightforward: if the data is masked out at a given index,
+we apply the operator, otherwise we'll continue to mask out the data.
+
+The available unary operators are:
+
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    abs
+    absolute
+    acos
+    arccos
+    acosh
+    arccosh
+    angle
+    asin
+    arcsin
+    asinh
+    arcsinh
+    atan
+    arctan
+    atanh
+    arctanh
+    bitwise_not
+    ceil
+    clamp
+    clip
+    conj_physical
+    cos
+    cosh
+    deg2rad
+    digamma
+    erf
+    erfc
+    erfinv
+    exp
+    exp2
+    expm1
+    fix
+    floor
+    frac
+    lgamma
+    log
+    log10
+    log1p
+    log2
+    logit
+    i0
+    isnan
+    nan_to_num
+    neg
+    negative
+    positive
+    pow
+    rad2deg
+    reciprocal
+    round
+    rsqrt
+    sigmoid
+    sign
+    sgn
+    signbit
+    sin
+    sinc
+    sinh
+    sqrt
+    square
+    tan
+    tanh
+    trunc
+
+The available inplace unary operators are all of the above **except**:
+
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    angle
+    positive
+    signbit
+    isnan
+
+Binary Operators
+----------------
+
+As you may have seen in the tutorial, :class:`MaskedTensor` also has binary operations implemented with the caveat
+that the masks in the two MaskedTensors must match or else an error will be raised. As noted in the error, if you
+need support for a particular operator or have proposed semantics for how they should be behave instead, please open
+an issue on Github. For now, we have decided to go with the most conservative implementation to ensure that users
+know exactly what is going on and are being intentional about their decisions with masked semantics.
+
+The available binary operators are:
+
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    add
+    atan2
+    arctan2
+    bitwise_and
+    bitwise_or
+    bitwise_xor
+    bitwise_left_shift
+    bitwise_right_shift
+    div
+    divide
+    floor_divide
+    fmod
+    logaddexp
+    logaddexp2
+    mul
+    multiply
+    nextafter
+    remainder
+    sub
+    subtract
+    true_divide
+    eq
+    ne
+    le
+    ge
+    greater
+    greater_equal
+    gt
+    less_equal
+    lt
+    less
+    maximum
+    minimum
+    fmax
+    fmin
+    not_equal
+
+The available inplace binary operators are all of the above **except**:
+
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    logaddexp
+    logaddexp2
+    equal
+    fmin
+    minimum
+    fmax
+
+Reductions
```

Comment:
I think we can add a section to this as well as a follow up.

Reply:
elaborated